tasks.py

```diff
@@ -70,19 +76,20 @@ def make_doc(ctx):
 
 
 @task
-def publish(ctx):
+def publish(ctx: Context) -> None:
     """
     Upload release to Pypi using twine.
 
-    :param ctx: Context
+    Args:
+        ctx (Context): The context.
     """
     ctx.run("rm dist/*.*", warn=True)
     ctx.run("python setup.py sdist bdist_wheel")
     ctx.run("twine upload dist/*")
 
 
 @task
-def set_ver(ctx, version):
```

Comment:
Can you please help me check this to make sure removal of unused arg doesn't break any workflow or anything? @janosh Thanks.

Reply:
`ctx` must be present in all `@task` decorated functions. see `invoke` docs. not sure we still use any of these functions but maybe @shyuep does and i know he likes to keep them around

Reply:
> `ctx` must be present in all `@task` decorated functions. see `invoke` docs. 

Thanks for the input.

> not sure we still use any of these functions but maybe @shyuep does and i know he likes to keep them around

Well I don't have any intention to remove them 😃